lms/djangoapps/support/static/support/templates/certificates.underscore

```diff
@@ -0,0 +1,10 @@
+
+<div class="certificates-search">
+    <form class="certificates-form">
+        <input type="text" name="query" value="" placeholder="<%- gettext("username or email") %>"></input>
```

Comment:
There should be a label associated with the input field (if it's not in the design it should be marked with class 'sr' so it is only read to screenreaders)